<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>291</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid orange;
border-radius:10px</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="textoInicio">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:orange;
font-size:20px;
border:none</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:orange;
font-size:15px;
border:none</string>
     </property>
     <property name="text">
      <string>Usuario</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:orange;
font-size:15px;
border:none</string>
     </property>
     <property name="text">
      <string>Senha</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_usuario">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:orange;
font-size:20px;
border:none;
border-radius:none;
border-bottom: 1px solid white;
color: white</string>
     </property>
     <property name="placeholderText">
      <string>Insira Seu nome de usuario</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2_senha">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:orange;
font-size:20px;

border:none;
border-radius:none;
border-bottom: 1px solid white;
color:white</string>
     </property>
     <property name="placeholderText">
      <string>Insira sua senha</string>
     </property>
    </widget>
    <widget class="QPushButton" name="botao1">
     <property name="geometry">
      <rect>
       <x>24</x>
       <y>312</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border:none;
border-radius:none;
background-color: rgb(255, 170, 0);</string>
     </property>
     <property name="text">
      <string>Logar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
border:none</string>
     </property>
     <property name="text">
      <string>Cadastre-se</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>101</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:none</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="pixmap">
      <pixmap>:/imgfile/OIP__1_-removebg-preview-removebg-preview.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_aviso">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: none;
color:red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>786</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../img/aaaa.qrc"/>
 </resources>
 <connections/>
</ui>
